app/src/main/java/ch/epfl/skysync/components/ConfirmFlightDetailUI.kt

```diff
@@ -119,6 +125,41 @@ fun ListElementText(text: String, padding: Dp, testTag: String) {
   )
   Spacer(modifier = Modifier.padding(4.dp))
 }
+
+@Composable
+fun HyperLinkText(title: String, value: String, padding: Dp) {
```

Comment:
Maybe calling it `location` rather than `value` would make more sense in this context